code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -24,11 +24,12 @@ const CourseProgress = React.createClass({
   propTypes: {
     professionalLearningCourse: React.PropTypes.bool,
     focusAreaPositions: React.PropTypes.arrayOf(React.PropTypes.number),
-    stages: React.PropTypes.arrayOf(stageShape)
+    stages: React.PropTypes.arrayOf(stageShape),
```

Comment:
Cool, no strong preference.  If we go with trailing commas let's enable the eslint rule to enforce it everywhere (separate PR).